picoCTF-web/api/routes/problem.py

```diff
@@ -139,3 +148,11 @@ def load_problems():
 
     api.problem.load_published(data)
     return WebSuccess("Inserted {} problems.".format(len(data["problems"])))
+
+
+@blueprint.route('/clear_submissions', methods=['GET'])
+@api_wrapper
+@require_login
```

Comment:
Do we also want to `@require_admin` too? This seems to be an admin-only operation, right?

Reply:
Correct, added.